coralproject/talk
# add a little keyboard shortcut hint widget

## What does this PR do?

in the moderation queue, there was no way to know that there were keyboard shortcuts. Now there's a little hint thing that can be dismissed letting you know that there's a wealth of shortcuts at your fingertips!

## How do I test this PR?

- view the moderation queue
- there should be a little shortcut thing there
- dismiss it
- reload the page; it should no longer be there
